tools/gulptasks/dashboards/scripts-dts/dashboards.src.d.ts

```diff
@@ -32,8 +33,13 @@ export { default as DataModifier } from "./es-modules/Data/Modifiers/DataModifie
 export { default as DataPool } from "./es-modules/Data/DataPool";
 export { default as DataTable } from "./es-modules/Data/DataTable";
 
-export const board: typeof Board.board;
-export const boards: typeof Globals.boards;
-export const win: typeof Globals.win;
+declare module Dashboards {
+    export const board: typeof Board.board;
+    export const boards: typeof Globals.boards;
+    export const win: typeof Globals.win;
+    export const error: typeof Utilities.error;
+    export const merge: typeof Utilities.merge;
+    export const uniqueKey: typeof Utilities.uniqueKey;
+}
 
-export as namespace Dashboards;
+export default Dashboards;
```

Comment:
This needs to be reverted as it breaks with classic setups, where the global namespace Dashboards is expected.